packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -977,10 +987,56 @@ export class ApexDebug extends LoggingDebugSession {
         this.myRequestService.connectionTimeoutMs =
           workspaceSettings.connectionTimeoutMs;
         break;
+      case EXCEPTION_BREAKPOINT_REQUEST:
```

Comment:
This is a request from the extension to the adapter for configuring an exception breakpoint.